http/src/main/resources/reference.conf

```diff
@@ -148,8 +148,15 @@ gce {
 }
 
 azure {
+   wsm {
+     uri = "https://localhost:8000"
+   }
+
    pubsub-handler {
     sam-url = ${sam.server}
+    wsm-url = ${azure.wsm.uri}
+    welder-acr-uri = "terradevacrpublic.azurecr.io/welder-server"
+    welder-image-hash = ${image.welderHash}
```

Comment:
hmm.. don't we have this in a specific place already?

I remember our CI that publishes welder hash changes to `reference.conf` had trouble last time we added another welder hash to the config. Could be worth looking into compatibility so we don't have to manually update this too...

Reply:
yea, it's referecing that exact config... which is why value here is ${image.welderHash}